packages/lab/src/calendar/internal/CalendarWeekHeader.css

```diff
@@ -19,3 +19,7 @@
   line-height: var(--calendar-week-header-size);
   font-size: var(--calendar-week-header-font-size);
 }
+
+.backwardsCompat.uitkCalendar {
+  --uitkCalendar-week-header-fontSize: var(--backwardsCompat-text-caption-fontSize);
```

Comment:
i'm not sure this is correct. wouldn't this override the api?

Reply:
that's a good point - i've changed all backwards compat classes (in this PR) to now reset local tokens, will make new PR for more changes needed as standardisation